lightwood/api/types.py

```diff
@@ -468,8 +376,10 @@ def from_dict(obj: Dict):
         """
         Creates a JSON-AI object from dictionary specifications of the JSON-config.
         """
-        features = {k: Feature.from_dict(v) for k, v in obj["features"].items()}
-        outputs = {k: Output.from_dict(v) for k, v in obj["outputs"].items()}
+        encoders = obj["encoders"]
+        dtype_dict = obj["dtype_dict"]
+        dependency_dict = obj.get('dependency_dict', None)
```

Comment:
default value `{}` instead?

Reply:
actually, I think the rule is "if it's not optional, no default value", in most cases, so I will stick to that